assets/chat/css/style.scss

```diff
@@ -82,6 +82,105 @@ label small {
     }
 }
 
+/* Generifys */
+// TODO maybe move this to anim.scss
+// TODO fix Hhhehehehe emote turning right for animation
+.mirror {
+    -moz-transform: scaleX(-1);
+    -o-transform: scaleX(-1);
+    -webkit-transform: scaleX(-1);
+    transform: scaleX(-1);
+}
+
+/*
+ * START WEATHER
+ * Minecraft weather effects with pure CSS
+ * © Copyright 2014 Moritz Marquardt
+ *
+ * https://github.com/moqmar/weather.css
+ *
+ * Licensed under MIT License (http://opensource.org/licenses/MIT)
```

Comment:
Note that as-per this license, we will need to include the permission notice found [here](https://github.com/moqmar/weather.css/blob/master/LICENSE) as well. Perhaps it could go in `LICENSE.md`?